content/en/upcoming-features.md

```diff
@@ -11,11 +11,13 @@ lastmod: 2020-02-20
 
 We are planning to add support for validating and including IP addresses in certificates.
 
+# Completed Features
+
 ## ECDSA Root and Intermediates
 
-Currently Let's Encrypt only signs end-entity certificates with RSA intermediates. Let's Encrypt will generate an ECDSA root and intermediates which can be used to sign end-entity certificates.
+* Enabled: September XX, 2020
```

Comment:
Let's not plan on incorporating this diff into the overall update, since we won't be issuing from the ECDSA intermediates right away. Alternately, we could update this to say: "We've generated an ECDSA hierarchy and will be working on getting it incorporated into root programs and beginning issuance from it."

Reply:
Ok, I've slowed the roll on this one :)